.eslintrc.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
```

Comment:
not a big deal, but this now doesn't have to be JS, it can be a nice simple static `.eslintrc` :-)

Reply:
Alright, nice and simple is great &mdash; SGTM.

It should probably be noted that once rules are added in the future, it may be a bit difficult to understand w/o comments.

Also, I've heard that not having extensions is an anti-pattern. Would you be agreeable to it being `.eslintrc.json` instead?

Reply:
`.eslintrc` supports JSON5, so it can have comments just fine.

It's fine if you want to include the extension; i never include it in my projects, and it feels incorrect to me to have JSON5 in a `.json` file.